examples/v2beta1/deepspeed/deepspeed-helloworld.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: deepspeed-helloworld
+spec:
+  slotsPerWorker: 1
+  runPolicy:
+    cleanPodPolicy: Running
+  mpiReplicaSpecs:
+    Launcher:
+      replicas: 1
+      template:
+        spec:
+          containers:
+          - image: registry.cn-beijing.aliyuncs.com/acs/deepspeed:hello-deepspeed
```

Comment:
It would be useful to check in the Dockerfile as well.

Reply:
You'er right. This demo is intend to be used to test the up coming feature, I'll manage it later.